src/GitHubUtil.js

```diff
@@ -101,19 +109,20 @@ async function getReviews(prNum) {
     let params = commonParams();
     params.pull_number = prNum;
 
-    const reviews = await GitHub.paginate(GitHub.rest.pulls.listReviews, params);
-    logApiResult(getReviews.name, params, {reviews: reviews.length});
-    // TODO: implement rateLimitedPromise() for 'paginated' results
-    return reviews;
+    const pages = await GitHub.paginate(GitHub.rest.pulls.listReviews, params, (response) => response);
```

Comment:
Do we need that no-op(?) lambda at the end? I see some paginate() [examples](https://github.com/octokit/octokit.js#pagination) that do not use it, but I am not sure I may be comparing apples to oranges. If it is needed, let's add a source code comment to document why.